tests/providers/google/marketing_platform/operators/test_campaign_manager.py

```diff
@@ -56,8 +56,7 @@ def test_execute(self, mock_base_op, hook_mock):
 
 class TestGoogleCampaignManagerGetReportOperator(TestCase):
     @mock.patch(
-        "airflow.providers.google.marketing_platform.operators."
-        "campaign_manager.http"
+        "airflow.providers.google.marketing_platform.operators." "campaign_manager.http"
```

Comment:
Can you revert these changes because they make it difficult for cherry-pick these operators to another project/package/release/etc.?

Reply:
Done